.github/workflows/build.yml

```diff
@@ -220,54 +222,73 @@ jobs:
         with:
           name: vim-macos-sha256sum.txt
           path: .
+      - id: shasum_check
+        run: |
+          curl -fsSL https://github.com/sasaplus1/vim/releases/latest/download/vim-linux-sha256sum.txt -o vim-linux-sha256sum-latest.txt
+          curl -fsSL https://github.com/sasaplus1/vim/releases/latest/download/vim-macos-sha256sum.txt -o vim-macos-sha256sum-latest.txt
+          if sha256sum --check vim-linux-sha256sum-latest.txt --check vim-macos-sha256sum-latest.txt
```

Comment:
This if condition returns false every time, because binary is include compiled date. 🤔